helm/templates/gateway/gateway-configmap.yaml

```diff
@@ -163,6 +163,75 @@ data:
       haproxy: {{ toYaml .Values.gateway.haproxy | nindent 8 }}
       {{- end }}
     {{- end }}
+    {{- if hasKey .Values.gateway "kafka" }}
+    kafka:
+      enabled: {{ .Values.gateway.kafka.enabled }}
```

Comment:
why not just using something like that?
`{{ toYaml .Values.gateway.kafka | indent 6 }}`

Because here, the "model" of the values.yml file fits exactly the "model" of the "gravitee.yml" file, doesn't it ?

Reply:
Yes, I thought about it. But i don't know, I don't really feel like we're facing it for the other places. The same goes for the TrustStore and KeyStore. 
We can discuss this with @jhaeyaert